src/ncorr_commandLine.cpp

```diff
@@ -526,10 +402,8 @@ int main(int argc, char *argv[]) {
 		interpType = INTERP::NEAREST;
 	}
 
-	numThreads = atof(
-		split(split(firstLevelSplit[4], "\n", false)[3], "\t")[1].c_str());
-	string subRegionstr;
-	subRegionstr = split(split(firstLevelSplit[4], "\n", false)[4], "\t")[1];

```

Comment:
Definitely good to get rid of this code

Reply:
Yeah, slightly unmaintainable. You can see why I decided to jump to json than figure this crap out again.